platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -510,6 +510,12 @@ This is currently used by the [JVM test suite plugin](userguide/jvm_test_suite_p
 
 See the [user manual](userguide/implementing_gradle_plugins_binary.html#custom_dependencies_blocks) to learn more about using this incubating feature.
 
+#### Testing Gradle 8.8 builds with TestKit using older Gradle version
```

Comment:
❌ We should not mention this in release notes since that's a potential breaking change. It belongs to the upgrade guide.